dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/DataOkHttpUploader.kt

```diff
@@ -30,7 +31,11 @@ internal abstract class DataOkHttpUploader(
             val response = call.execute()
             responseCodeToUploadStatus(response.code())
         } catch (e: Throwable) {
-            sdkLogger.e("Unable to upload batch data", e)
+            sdkLogger.e(
+                "Unable to upload batch data. Executor active threads:" +
+                    " [${CoreFeature.uploadExecutorService.activeCount}]",
```

Comment:
Can you make this an attribute please instead? That way we'll be able to filter, query and graph it.